src/main/scala/org/phenoscape/kb/PresenceAbsenceOfStructure.scala

```diff
@@ -62,11 +62,14 @@ object PresenceAbsenceOfStructure {
 
   def presenceAbsenceMatrix(mainEntityClass: OWLClassExpression, taxonClass: OWLClassExpression, variableOnly: Boolean, includeParts: Boolean): Future[DataSet] = {
     val entityClass = if (includeParts) mainEntityClass or (part_of some mainEntityClass) else mainEntityClass
+    val buildDateFut = KB.buildDate
     for {
       query <- App.expandWithOwlet(buildMatrixQuery(entityClass, taxonClass))
       model <- App.executeSPARQLConstructQuery(query)
+      buildDate <- buildDateFut
     } yield {
       val dataset = new DataSet()
+      dataset.setCurators("Ontotrace matrix generated from Phenoscape KB built on " + buildDate.toString)
```

Comment:
_Ontotrace matrix generated from Phenoscape KB built on DD-Mon-YYYY_ isn't really an author, but is a provenance statement in natural language. Wouldn't it be more appropriate to say _Phenoscape KB built on  DD-Mon-YYYY_?

Reply:
I agree.